index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Editor</title>
+    
+    <style type="text/css" media="screen">
```

Comment:
Why don't you use for external CSS?

Reply:
@juleshwa I probably would have eventually. The Ace implementation starting code used the styling inside of the html page. Using DOM, I planned on overriding external css styling within the html page.